scss/1-settings/_colour-language.scss

```diff
@@ -1,62 +1,29 @@
 @use '../1-settings/colour-palette' as palette;
+// ${component}__{element}__{property}--{state}
+$cads-language__brand--primary: palette.$cads-palette__heritage-blue !default;
+$cads-language__brand--primary-dark: palette.$cads-palette__heritage-blue-dark !default;
+$cads-language__brand--secondary: palette.$cads-palette__heritage-yellow !default;
+$cads-language__brand--primary-contrast: palette.$cads-palette__white !default;
+$cads-language__brand--adviser: palette.$cads-palette__blue-adviser !default;
+$cads-language__brand--focus: palette.$cads-palette__yellow !default;
```

Comment:
I'm not sure I want to call it `brand--focus` because I don't know if fits `brand` naming. However, naming the colour just `$cads-language__focus` also doesn't feel quite right as focus is not an element. Ideas are welcome